pr-dhl-woocommerce/includes/REST_API/Parcel_DE/Item_Info.php

```diff
@@ -691,7 +691,16 @@ protected function get_post_office_address_schema() {
 					return $self->country_code_to_alpha3( $countryCode );
 				},
 			),
-			'email'       => array(),
+			'email'       => array(
+				'error'    => esc_html__( 'Postfiliale needs post number or email', 'dhl-for-woocommerce' ),
+				'validate' => function ( $email, $args ) {
+					$has_postnum = ! empty( $args['dhl_postnum'] );
```

Comment:
This will likely cause an error. I believe you intended to do something like this instead:
```suggestion
				'validate' => function ( $email ) use ( $self ) {
					$has_postnum = ! empty( $self->packStationAddress['dhl_postnum'] );
```